councilmatic_core/migrations/0048_post_shape.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.9 on 2019-06-26 17:10
+
+import django.contrib.postgres.fields.jsonb
+import django.core.files.storage
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0004_auto_20171005_2028'),
+        ('councilmatic_core', '0047_update_filepath'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Post',
+        ),
+        migrations.AlterField(
+            model_name='person',
+            name='headshot',
+            field=models.FileField(default='images/headshot_placeholder.png', storage=django.core.files.storage.FileSystemStorage(base_url='/', location='/Users/goobzie/datamade/chi-councilmatic'), upload_to='images/headshots'),
+        ),
```

Comment:
Seems like this migration didn't get caught earlier. It also has a path specific to my machine, since the `storage` param is set dynamically. If it'll pose a problem, I can remove this from the migration and we can deal with it in a separate PR.

Reply:
this does not seem like what we want. Let's deal with headshots in a separate PR.